Purchases/Support/BeginRefundRequestHelper.swift

```diff
@@ -121,6 +121,13 @@ private extension BeginRefundRequestHelper {
                     return
                 }
 
+                guard customerInfo.entitlements.active.count < 2 else {
+                    let message = Strings.purchase.begin_refund_multiple_active_entitlements.description
+                    continuation.resume(throwing: ErrorUtils.beginRefundRequestError(withMessage: message))
+                    Logger.error(message)
```

Comment:
We've talked about this. It still needs work, but for the time being we might want to avoid double logging it?